MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+include LICENSE
+include *.md
+recursive-include tests/ *.py *.md *.json
+graft man
+graft cvelib/schemas
+global-exclude __pycache__
+global-exclude *.py[co]
```

Comment:
Not quite sure what this file is doing - did it get committed by mistake?

Reply:
Ah, I bet this logic controls which files end up in the built package. LGTM then.

Reply:
Correct, when building a binary or source package (`python3 setup.py sdist bdist`) the default is to only include Python files from the package identified in `setup.py`'s `packages` attribute. Everything else has to be included explicitly in a MANIFEST.in file.

I stole the exclude lines from [Django's manifest](https://github.com/django/django/blob/main/MANIFEST.in) because the pycache files were being added as part of the tests/ directory for some reason.